server/eb/app.py

```diff
@@ -80,8 +79,13 @@ def _before_adding_routes(app, app_config):
         web_base_url = server_config.get_web_base_url()
         if web_base_url:
             web_base_url_parse = urlparse(web_base_url)
-            allowed_origin = f"{web_base_url_parse.scheme}://{web_base_url_parse.netloc}"
-            CORS(app, supports_credentials=True, origins=allowed_origin)
+            allowed_origins = [f"{web_base_url_parse.scheme}://{web_base_url_parse.netloc}"]
+            if os.getenv('DEPLOYMENT_STAGE') in ["Staging"]:
```

Comment:
Ah I found [here](https://github.com/chanzuckerberg/single-cell-infra/blob/17047ab5bcfdff5a6e1c3e62eeb791628cf83c01/terraform/envs/staging/dp-staging/variables.tf#L2) uses `staging` instead of `Staging`, but not sure if that's where we're getting `DEPLOYMENT_STAGE` from 😆

Reply:
I tested by deploying to staging and "Staging" seemed to work but I can include both for safety

Reply:
```suggestion
            if os.getenv('DEPLOYMENT_STAGE') in ["Staging", "staging"]:
```